### PR TITLE
Add GoToMeeting connector

### PR DESCRIPTION
**Pull request by Chuhaa**
## Purpose
$subject

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Test environment
JDK 11
Ballerina SL Beta3
Ubuntu 20.04

### CODE REVIEW

File: openapi/goto.meeting/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[package]
+org = "ballerinax"
+name = "goto.meeting"
```
Comment (indikasampath2000) [inline openapi/goto.meeting/Ballerina.toml:3]: I think `gotomeeting` should be a single word. can you update the module folder name as well?

Comment (Chuhaa) [reply openapi/goto.meeting/Ballerina.toml:3]: But GoTo provides four APIs [GoToMeeting](https://developer.goto.com/GoToMeetingV1), [GoToWebinar](https://developer.goto.com/GoToWebinarV2), [GoToTraining](https://developer.goto.com/GoToTrainingV1) and [GoToConnect](https://developer.goto.com/GoToConnect). Multiple APIs from a vendor.

Comment (indikasampath2000) [reply openapi/goto.meeting/Ballerina.toml:3]: It's a bit wired to have a dot separation here. Let's have names like `gotomeeting` `gotowebinar` `gototraining` `gotoconnect`

Comment (Chuhaa) [reply openapi/goto.meeting/Ballerina.toml]: Ack. Fixed in https://github.com/ballerina-platform/ballerinax-openapi-connectors/commit/fa14730cbafb911593c56574c6a8d825030f2546

**Review by indikasampath2000 — APPROVED** [review]

**Pull request merged**